### PR TITLE
add react-debounce-input

### PR DESCRIPTION
**Pull request by darkleaf**
New package:

**Extern:** Generated.

### CODE REVIEW

File: react-debounce-input/build.boot [hunk 1]
```diff
@@ -0,0 +1,30 @@
+(def +lib-version+ "3.0.0")
+(def +version+ (str +lib-version+ "-0"))
+
+(set-env!
+ :resource-paths #{"resources"}
+ :dependencies   [['cljsjs/boot-cljsjs "0.5.2" :scope "test"]])
```
Comment (Deraen) [inline react-debounce-input/build.boot]: In addition to deps-cljs react require, this package should also have a dependency on `cljsjs/react`.

Comment (darkleaf) [inline react-debounce-input/build.boot]: Why?

My scenario.
I use previous version of react in my app.
I added cljsjs/react dep in my first commit in this PR.
But my app began to use new version form cljsjs/react-debounce-input.
And I wrote `[cljsjs/react-debounce-input "3.0.0-0" :exclusions [cljsjs/react]]`.

react-autosuggest [has no dependency from react](https://github.com/cljsjs/packages/blob/master/react-autosuggest/build.boot#L3).

Comment (Deraen) [inline react-debounce-input/build.boot]: To ensure React is present, in a rare case where there is no other dependency on it. If you need specific version you should add exclusions or direct dependency.

https://github.com/cljsjs/packages/wiki/Foreign-library-dependencies

There are many bad packages as I don't always notice all the problems.

Comment (darkleaf) [inline react-debounce-input/build.boot]: @Deraen I revert my commit.

**Pull request merged**